content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx

```diff
@@ -0,0 +1,75 @@
+import responseJson from '../_examples/response';
+
+<Schema
+  json={responseJson.ip_address}
+  jsonPointer="/ip_address"
+  name="Response | IP Address"
+>
+  For minFraud Score, this object only contains the `risk` for the IP address.
+  For minFraud Insights and Factors, the object is the
+  [GeoIP2 Insights response body](#) with four modifications:
```

Comment:
The text says there are 4 modifications but only 3 are numbered. I think `minFraud Insights and Factors return the following anonymous IP outputs` should be #4?

Reply:
Huh. Take a look at this - https://dev.maxmind.com/minfraud/#IP_Address_ip_address

Perhaps it's supposed to be "three modifications" on the current dev site too?

Reply:
yes, that would make sense.